proto/skaffold.proto

```diff
@@ -344,11 +353,32 @@ enum ErrorCode {
     STATUS_CHECK_NODE_NOT_READY= 406;
 
     // Unknown Error Codes
+    ErrorCode_UNKNOWN = 501;
 
     // Status Check error unknown
-    STATUS_CHECK_UNKNOWN = 501;
+    STATUS_CHECK_UNKNOWN = 502;
     // Container is unschedulable due to unknown reasons
-    STATUS_CHECK_UNKNOWN_UNSCHEDULABLE = 502;
+    STATUS_CHECK_UNKNOWN_UNSCHEDULABLE = 503;
     // Container is waiting due to unknown reason
-    STATUS_CHECK_CONTAINER_WAITING_UNKNOWN = 503;
+    STATUS_CHECK_CONTAINER_WAITING_UNKNOWN = 504;
+
+
+
+    // FILE_SYNC Failures
+
+    // File Sync Initialize failure
+    FILE_SYNC_INIT_ERROR = 601;
```

Comment:
Do we need the _ERROR suffix?  Isn't it implicit?

Reply:
dropped

Reply:
Change this to FILE_SYNC_INIT_FAIL